stacks/apps/apple-api-bridge.yml

```diff
@@ -0,0 +1,349 @@
+# stacks/apps/apple-api-bridge.yml
+# 100A
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+
+Description: A serverless API application to batch api requests to Apple
+
+Conditions:
+  IsProduction: !Equals [!Ref EnvironmentType, Production]
+
+Parameters:
+  EnvironmentType: { Type: String }
+  EnvironmentTypeAbbreviation: { Type: String }
+  RootStackName: { Type: String }
+  RootStackId: { Type: String }
+  CloudWatchAlarmTaggerServiceToken: { Type: String }
+  CloudWatchLogGroupTaggerServiceToken: { Type: String }
+  CodeS3Bucket: { Type: String }
+  CodeS3ObjectKey: { Type: AWS::SSM::Parameter::Value<String> }
+
+Resources:
+  HttpApi:
+    Type: AWS::Serverless::HttpApi
+    Properties:
+      StageName: !Ref EnvironmentType
+      Description: !Sub >-
+        ${EnvironmentType} Apple Api Bridge API
+      Tags:
+        prx:meta:tagging-version: "2021-04-07"
+        prx:cloudformation:stack-name: !Ref AWS::StackName
+        prx:cloudformation:stack-id: !Ref AWS::StackId
+        prx:cloudformation:root-stack-name: !Ref RootStackName
+        prx:cloudformation:root-stack-id: !Ref RootStackId
+        prx:ops:environment: !Ref EnvironmentType
+        prx:dev:family: Dovetail
+        prx:dev:application: Apple Api Bridge
+
+  AppleApiBridgeLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri:
+        Bucket: !Ref CodeS3Bucket
+        Key: !Ref CodeS3ObjectKey
+      Handler: app.lambdaHandler
+      Runtime: nodejs18.x
+      Description: !Sub ${EnvironmentType} Apple API Bridge
+      MemorySize: 128
+      Architectures:
+        - x86_64
```

Comment:
If there are no binary dependencies for this app, I would use the `arm64` architecture. It tends to be more resource efficient.